docs/api/nn/pool.md

```diff
@@ -53,3 +53,25 @@
         members:
             - __init__
             - __call__
+
+---
+
+::: equinox.nn.adaptive_avg_pool1d
```

Comment:
I think we can leave this out of the docs, it's really just a private helper function.